models/Playground.js

```diff
@@ -3,8 +3,7 @@ const { Schema } = mongoose;
 
 const playGroundSchema = new Schema({
   checkedIn: {
-    type: Number,
-    default: 0,
+    type: [String],
```

Comment:
why is String here in Brackets and not in line 9?

Reply:
because its a mongoose thing.. it says, everything inside an array should be a string.

Reply:
ouh nice!